src/components/Board.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useState, useEffect } from 'react'
+import Square from './Square'
+import lines from './winning-conditions'
+import Confetti from 'react-dom-confetti'
+import confettiConfig from './confetti-config'
+
+//Board props
+type BoardType = {
+  squares: string[][]
+  updateHistory: (squares: string[]) => void
+  currTurn: string
+}
+
+//component
+const Board: React.FC<BoardType> = (props: BoardType) => {
```

Comment:
```suggestion
const Board = ({ squares, updateHistory, currTurn }: BoardType) => {
```

always just destructure props when creating React components

you'll save a lot of time not writing props. for everything